src/houses/KM_HouseSchool.pas

```diff
@@ -218,7 +268,7 @@ function TKMHouseSchool.QueueIsEmpty: Boolean;
 
 function TKMHouseSchool.QueueCount: Byte;
 var
-  I: Integer;
+  I: Byte;
```

Comment:
It is better to stick to using Integers for loops, since High(something) could be -1 and being cast to uint it becomes maxUInt-1.